manifests/profiles/ambient.yaml

```diff
@@ -36,6 +36,7 @@ spec:
       privileged: true
       ambient:
         enabled: true
+        redirectMode: "ebpf"
```

Comment:
```suggestion
```
Sorry, last minute thing - 

Can I suggest we leave this _off_ for now, and continue defaulting to the existing iptables mode - making eBPF an opt-in on top of the opt-in for ambient?


Mostly because

1. Ambient is in `master` now and I would rather not destabilize it further by default.
2. This has had less testing than the existing mode, and has some constraints the existing one doesn't.
3. Doesn't work out of the box for me on e.g. `kind` where the iptables mode does.

Reply:
oops, good call. I agree, thought it was off

Reply:
+1 - I would prefer to keep iptables as default as that is well tested.

Reply:
done